### PR TITLE
Add some information about QtMir and QtUbuntu

### PR DESCRIPTION
**Pull request by UniversalSuperBox**
Nothing terribly exciting, just a bunch of links and some information about getting verbose logging. It helped me a lot to find these, so hopefully it'll help future goers.

### CODE REVIEW

File: systemdev/qtmir-qtubuntu.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+QtMir and QtUbuntu
+==================
+
+`QtMir`_ and `QtUbuntu`_ are `Qt Platform Abstractions`_ (QPAs) for the Ubuntu Touch platform. This page is a collection of notes on debugging the QPAs and their related pieces.
+
+QtMir
+-----
+
+The `QtMir`_ QPA makes Qt a Mir server. This allows Unity8, the shell of Ubuntu Touch, to be written in QML and Qt C++ while operating on Mir Surfaces and Windows. It also allows the desktop to be represented and reasoned about as a `Qt Scene Graph`_.
```
Comment (doniks) [inline systemdev/qtmir-qtubuntu.rst]: Does it really "make Qt a Mir server"? That sounds odd. Isn't it more like .... provides Qt bindings for Mir or exposes Mir API or smt like that?

File: systemdev/qtmir-qtubuntu.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+You'll find most of QtMir's logging in Unity8's logs, normally stored at ``/home/phablet/.cache/upstart/unity8`` on Ubuntu Touch. Different logging categories, like ``qtmir.sessions`` and ``qtmir.surfaces``, relate directly to different modules within QtMir's codebase.
+
+While Unity8 shows almost all QtMir logging by default, you can enable even more verbose logging by placing the following content in the file ``/usr/share/upstart/sessions/unity8.override`` and restarting Unity8:
```
Comment (doniks) [inline systemdev/qtmir-qtubuntu.rst]: maybe add the command for restarting? smt with `initctl` or `service`?

File: systemdev/qtmir-qtubuntu.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
+QtMir and QtUbuntu
+==================
+
+`QtMir`_ and `QtUbuntu`_ are `Qt Platform Abstractions`_ (QPAs) for the Ubuntu Touch platform. This page is a collection of notes on debugging the QPAs and their related pieces.
```
Comment (doniks) [inline systemdev/qtmir-qtubuntu.rst]: Can you add a sentence about what's the difference? Why do we have two QPAs?

File: systemdev/qtmir-qtubuntu.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+While Unity8 shows almost all QtMir logging by default, you can enable even more verbose logging by placing the following content in the file ``/usr/share/upstart/sessions/unity8.override`` and restarting Unity8:
+
+.. code-block:: sh
+
+    env QT_LOGGING_RULES='qtmir.*=true'
+
+QtUbuntu
+--------
+
+The `QtUbuntu`_ QPA uses the Mir client APIs and `Ubuntu Platform API`_ to provide Ubuntu Touch apps with a stable hardware compatibility API.
```
Comment (doniks) [inline systemdev/qtmir-qtubuntu.rst]: link the Mir client APIs?

File: systemdev/qtmir-qtubuntu.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    initctl set-env QT_LOGGING_RULES='ubuntumirclient.*=true'
+
+We plan to replace QtUbuntu with QtWayland in the future.
+
+Components
+^^^^^^^^^^
+
+QtUbuntu has a number of sub-components to provide other features in the Ubuntu Touch platform. In theory, these could be used on other platforms to provide similar features. In practice, this has never happened.
+
+`QtUbuntu-Camera`_ provides the ``aalCamera`` (Android Abstraction Layer Camera) plugin to QtMultimedia. This plugin allows apps to access Android device cameras through the QtMultimedia standard API. We are trying to replace this component with the `gst-droid`_ plugin for GStreamer for all new Android device ports.
```
Comment (doniks) [inline systemdev/qtmir-qtubuntu.rst]: link QtMultimedia

**Comment by UniversalSuperBox** [comment]
Excellent points. I think I've addressed all of them.

**Comment by UniversalSuperBox** [comment]
Some documentation is better than all the documentation, and this is all I have time for right now.

**Pull request merged**